<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FUNCIONARIOS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>521</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Funcionário </string>
    </property>
    <widget class="QLineEdit" name="txtNome">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>49</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>45</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCargo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cargo: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRg">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>############</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCpf">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>############</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDataNascimento">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>##/##/####</string>
     </property>
     <property name="text">
      <string>//</string>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCelular">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>330</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>##-#####-####</string>
     </property>
     <property name="maxLength">
      <number>13</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCidade">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>410</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCep">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>450</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>#####-###</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RG: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CPF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cep</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cidade </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Logradouro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>330</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Celular</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Telefone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Data de Nascimento </string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>580</y>
       <width>521</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEntrada">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>530</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>##:##</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>500</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Salário</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>540</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Entrada</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>580</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Saída</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCadastrarfuncionario">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>640</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Cadastrar </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnListarfuncionarios">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>640</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Listar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTelefone">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>##-####-####</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSalario">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>490</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>####,##</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLogradouro">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSaida">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>570</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>##:##</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSobrenome">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>45</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>91</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sobrenome</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtNome</tabstop>
  <tabstop>txtSobrenome</tabstop>
  <tabstop>txtCargo</tabstop>
  <tabstop>txtRg</tabstop>
  <tabstop>txtCpf</tabstop>
  <tabstop>txtDataNascimento</tabstop>
  <tabstop>txtTelefone</tabstop>
  <tabstop>txtCelular</tabstop>
  <tabstop>txtLogradouro</tabstop>
  <tabstop>txtCidade</tabstop>
  <tabstop>txtCep</tabstop>
  <tabstop>txtSalario</tabstop>
  <tabstop>txtEntrada</tabstop>
  <tabstop>txtSaida</tabstop>
  <tabstop>btnCadastrarfuncionario</tabstop>
  <tabstop>btnListarfuncionarios</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
